website/public/index.ejs.html

```diff
@@ -802,6 +802,128 @@ <h2 id="linter-feature-comparison">Features</h2>
+            <tr class="header">
+              <th class="quick-lint-js">quick-lint-js</th>
+              <th><a href="https://eslint.org/">ESLint</a></th>
+              <th><a href="https://jshint.com/">JSHint</a></th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td class="good quick-lint-js has-link">
+                <a href="benchmarks/"
+                  ><span aria-labelledby="feature-speed-qljs-quick">🚀</span
+                  ><br /><span id="feature-speed-qljs-quick">quick</span></a
+                >
+              </td>
+              <td class="meh">
+                <span aria-labelledby="feature-speed-eslint-capable">🐇</span
+                ><br /><span id="feature-speed-eslint-capable">capable</span>
+              </td>
+              <td class="bad">
+                <span aria-labelledby="feature-speed-jshint-slow">🐢</span
+                ><br /><span id="feature-speed-jshint-slow">slow</span>
+              </td>
+            </tr>
+            <tr>
+              <td class="good quick-lint-js">
+                <span aria-label="easy">✅</span><br />zero config
+              </td>
+              <td class="meh">
+                <span aria-label="hard">❌</span><br />config or<br />
+                <code>init</code>
+              </td>
+              <td class="bad"><span aria-label="hard">❌</span><br />config</td>
+            </tr>
+
+            <tr>
+              <td class="good quick-lint-js">free<br />GPLv3</td>
+              <td class="meh">
+                open<br />
+                source<br />MIT
+              </td>
+              <td class="meh">
+                open<br />
+                source<br />MIT
+              </td>
+            </tr>
+            <tr>
+              <td class="good quick-lint-js">
+                <span aria-label="yes">✅<br />JSX</span>
+              </td>
+              <td class="meh">
+                <span aria-label="yes">✅<br />JSX</span>
+              </td>
+              <td class="bad">
+                <span aria-label="no">❌<br />JSX</span>
+              </td>
+            </tr>
+            <tr>
+              <td class="bad quick-lint-js">
+                <span aria-label="no">❌<br />TypeScript</span>
+              </td>
+              <td class="meh">
+                <span aria-label="yes">✅<br />TypeScript</span>
+              </td>
+              <td class="bad">
+                <span aria-label="no">❌<br />TypeScript</span>
+              </td>
+            </tr>
+
+            <tr>
+              <td class="meh quick-lint-js">
+                <a href="errors/">minimal</a><br />Lint Rules
+              </td>
+              <td class="good">
+                <a href="https://eslint.org/docs/rules/">zillions</a><br />Lint
+                Rules
+              </td>
+              <td class="meh">
+                <a href="https://jshint.com/docs/options/">some</a><br />Lint
+                Rules
```

Comment:
I'd adjust the capitalization and make the phases grammatically correct:
```suggestion
                <a href="errors/">minimal</a><br />lint rules
              </td>
              <td class="good">
                <a href="https://eslint.org/docs/rules/">zillions</a><br />of lint
                rules
              </td>
              <td class="meh">
                <a href="https://jshint.com/docs/options/">some</a><br />lint
                rules
```